File: hash-generator.py
import hashlib
import os
# Simulasi user agent (ambil dari Termux environment)
ua = "Mozilla/5.0 (Linux; Android 12; Mobile) AppleWebKit/537.36"
# Layar default HP Android (asumsi 412x915)
screen_info = "412x915-24"
lang = "id-ID"
raw = ua + screen_info + lang
hash_bytes = hashlib.sha256(raw.encode()).hexdigest()
# Ambil 8 karakter pertama dari SHA256 untuk konsistensi (bisa disesuaikan)
short_hash = hash_bytes[:8].upper()
print("DEV-" + short_hash)

File: hash-generator.js
// Fungsi yang sama dengan di HTML
const ua = "Mozilla/5.0 (Linux; Android 12; Mobile)"; // kamu bisa ganti dengan user agent asli jika perlu
const screenInfo = screen ? `${screen.width}x${screen.height}-${screen.colorDepth}` : "412x915-24";
const raw = ua + screenInfo + (navigator ? navigator.language : "id-ID");
let h = 0;
for (let i = 0; i < raw.length; i++) {
    h = ((h << 5) - h) + raw.charCodeAt(i);
    h |= 0;
}
console.log('DEV-' + Math.abs(h).toString(16).toUpperCase().slice(0,8));
